module-3_Postgresql/src/middleware/logger-handling.middleware.ts

```diff
@@ -0,0 +1,14 @@
+import { NextFunction, Response, Request } from 'express';
+import log from '../logger';
+
+export const loggerHandlerMiddleware = (req: Request, res: Response, next: NextFunction) => {
+    const { path, method, params, query, body } = req;
+    const metadata = {
+        method,
+        path,
+        args: params || query || {},
+        body
```

Comment:
You should make sure you not log User sensitive information like passwords etc. So you may add check if password exists in input body and obscure it.